index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="utf-8">
+  <title>My Portfolio</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ </head>
+ <body>
+  <header>
+   <div class="icon"><p>My Logo</p></div>
+   <img src="images/Union.png" alt="">
+   <link rel="stylesheet" href="style.css">
+  </header>
+  <main>
+   <article class="about">
+    <h1>I’m Mariane
+     Glad to see you!</h1>
+    <p>I’m a software developer! I can help you build a product , 
+     feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <a href="">LET’S CONNECT</a>
+   </article>
+   <article class="social_media">
+    <img src="images/twiter.svg" alt="">
+    <img src="images/link.svg" alt="">
+    <img src="images/vector.svg" alt="">
+    <img src="images/vector1.svg" alt="">
+    <img src="images/vector2.svg" alt="">
```

Comment:
- [ ] lets make each of our individual icons to look like this `<li> <a href=""><img src="images/twiter.svg" alt=""></a></li>`
 as required in the project requirement. this will help making the icons clickable for easy page navigation